rtloader/include/rtloader_types.h

```diff
@@ -121,6 +121,10 @@ typedef void (*cb_log_t)(char *, int);
 typedef void (*cb_set_check_metadata_t)(char *, char *, char *);
 // (hostname, source_type_name, list of tags)
 typedef void (*cb_set_external_tags_t)(char *, char *, char **);
+// (key, value)
+typedef void (*cb_write_persistent_cache_t)(char *, char *);
+// (value)
+typedef void (*cb_read_persistent_cache_t)(char *, char **);
```

Comment:
We can return a `char*` in CGO and I still think it makes more sens to return the value

Reply:
I trust you on that one, but it seems weird that none of the other functions do so.